p11-kit/add-profile.c

```diff
@@ -106,9 +106,8 @@ add_profile (const char *token_str,
 	}
 
 	p11_kit_iter_begin (iter, modules);
-	rv = p11_kit_iter_next (iter);
-	if (rv != CKR_OK) {
-		p11_message (_("failed to find the token: %s"), p11_kit_strerror (rv));
+	if (p11_kit_iter_next (iter) != CKR_OK) {
```

Comment:
I guess we could special case `CKR_CANCEL` in this case, as there might be other errors?